packages/grid/src/index.scss

```diff
@@ -8,7 +8,26 @@
 $grid-gutter-width: 20px !default;
 
 @import '@zendeskgarden/css-variables/dist/index.scss';
-@import 'bootstrap/scss/bootstrap-grid';
+
+/**
+ * Bootstrap grid imports
+ */
+@import 'bootstrap/scss/functions';
+@import 'bootstrap/scss/variables';
+@import 'bootstrap/scss/mixins/breakpoints';
+@import 'bootstrap/scss/mixins/grid-framework';
+@import 'bootstrap/scss/mixins/grid';
+@import 'bootstrap/scss/grid';
+@import 'bootstrap/scss/utilities/display';
+@import 'bootstrap/scss/utilities/flex';
+
+.container, .container-fluid {
+  *,
+  *::before,
+  *::after {
+    box-sizing: inherit;
+  }
+}
```

Comment:
Unlike the previous dist, this fails to apply `box-sizing` directly to the container classes.

Reply:
Now added.